msticpyconfig.yaml

```diff
@@ -22,11 +24,11 @@ TIProviders:
   # # (e.g. XForce requires an ApiID key as well). The "AuthKey" name
   # # will be mapped onto the name used by the service, you should not
   # # to change this - e.g. some providers call this ApiKey. 
-  # OTX:
-  #   Args:
-  #     AuthKey: OTX-API-Key
-  #   Primary: True
-  #   Provider: "OTX"
+   OTX:
+     Args:
+       AuthKey: "be062f0a69fc7732acaf5eb6e95d55c125b7f0ecc1d349cb28d4a715d67782e9"
```

Comment:
You should get rid of this key and regenerate a new one in OTX
KeyVault is a better place to keep these than github :-D

Reply:
oops, will do! Thanks for letting me know!